staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -867,30 +868,20 @@ func (r *crdHandler) getOrCreateServingInfoFor(crd *apiextensionsv1.CustomResour
 			}
 		}
 
-		columns, err := getColumnsForVersion(crd, v.Name)
-		if err != nil {
-			utilruntime.HandleError(err)
-			return nil, fmt.Errorf("the server could not properly serve the CR columns")
-		}
-		table, err := tableconvertor.New(columns)
-		if err != nil {
-			klog.V(2).Infof("The CRD for %v has an invalid printer specification, falling back to default printing: %v", kind, err)
+		var table rest.TableConvertor
+		if r.tableConverterProvider != nil {
+			table = r.tableConverterProvider.GetTableConverter(crd.Spec.Group, v.Name, crd.Status.AcceptedNames.Kind, crd.Status.AcceptedNames.ListKind)
```

Comment:
```
# k8s.io/apiextensions-apiserver/pkg/apiserver                                                                                                                                                                                                
../../kubernetes/kubernetes/staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go:873:54: too many arguments in call to r.tableConverterProvider.GetTableConverter                                               
        have (string, string, string, string)                                                                                                                                                                                                 
        want (string, string, string)    
```

looks like the function takes `group, kind, listKind string`, you're passing `crd.Status.AcceptedNames.Kind,` as well

Reply:
Thanks, fixing